mp/src/game/client/hud_crosshair.cpp

```diff
@@ -229,6 +272,249 @@ void CHudCrosshair::GetDrawPosition ( float *pX, float *pY, bool *pbBehindCamera
 	*pbBehindCamera = bBehindCamera;
 }
 
+void CHudCrosshair::DrawCrosshair( CWeaponBase *weaponBase )
+{
+    if ( !IsCurrentViewAccessAllowed() )
+		return;
+    /*
+	C_BasePlayer* pPlayer = C_BasePlayer::GetLocalPlayer();
+	if ( !pPlayer )
+		return;
+    */
+
+	const auto pPlayer = C_MomentumPlayer::GetLocalMomPlayer();
+
+    if (!pPlayer)
+        return;
+
+    // localplayer must be owner if not in Spec mode
+    Assert((pPlayer == weaponBase->GetPlayerOwner()) || (pPlayer->GetObserverMode() == OBS_MODE_IN_EYE));
+
+    // Draw the targeting zone around the pCrosshair
+    if (pPlayer->IsInVGuiInputMode())
+        return;
+
+	float x, y;
+	bool bBehindCamera;
+    GetDrawPosition(&x, &y, &bBehindCamera, m_vecCrossHairOffsetAngle);
+    
+	if( bBehindCamera )
+		return;
```

Comment:
Looks like spacing is off, after you fix the compile for Linux can you format this file to use spaces?